packages/compass-query-bar/src/components/option-editor.tsx

```diff
@@ -97,7 +97,7 @@ type OptionEditorProps = {
 export const OptionEditor: React.FunctionComponent<OptionEditorProps> = ({
   id,
   hasError = false,
-  hasAutofix = false,
+  hasAutofix = true,
```

Comment:
Super NIT: i know this is just me and that i am the usual pain .. should we rename this to `autoinsertEmptyDocument` (acknowleging the language you used in the PR title to communicate the same thing) .. or something that would hint to what this autofixing is doing?

Reply:
Renamed to `autoinsertEmptyDocOnFocus` in https://github.com/mongodb-js/compass/pull/5135/commits/3c2731adc106616b03591a883004b31631281685